mozilla_workgroup/README.md

```diff
@@ -1,76 +1,50 @@
-# Mozilla Workgroup
+# workgroup
+Retrieve workgroup ACL lists associated with data and gcp access workgroups.
 
-Builds on top of the [google workgroup module](../google_workgroup/) and contains constants for mozilla environments. This is the default to use for creating our internal tenants.
+Workgroup identifiers should be of the form:
 
-<!-- START_TF_DOCS -->
-## Requirements
+```
+workgroup:WORKGROUP_NAME[/SUBGROUP]
+```
 
-| Name | Version |
-|------|---------|
-| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | ~> 1.0 |
-| <a name="requirement_google"></a> [google](#requirement\_google) | >= 3.0 |
-| <a name="requirement_google-beta"></a> [google-beta](#requirement\_google-beta) | >= 4.0 |
+where `SUBGROUP` defaults to `default`. For example: `workgroup:app`, `workgroup:app/admin`.
 
-## Providers
+For subgroup queries across all workgroups, an additional identifier format:
 
-No providers.
+```
+subgroup:SUBGROUP
+```
 
-## Modules
+is supported, which will return all workgroups that contain a particular subgroup.
 
-| Name | Source | Version |
-|------|--------|---------|
-| <a name="module_workgroup"></a> [workgroup](#module\_workgroup) | github.com/mozilla/terraform-modules//google_workgroup | OPST-682 |
-
-## Resources
-
-No resources.
-
-## Inputs
-
-| Name | Description | Type | Default | Required |
-|------|-------------|------|---------|:--------:|
-| <a name="input_ids"></a> [ids](#input\_ids) | List of workgroup identifiers to look up access for | `set(string)` | n/a | yes |
-
-## Outputs
-
-| Name | Description |
-|------|-------------|
-| <a name="output_google_groups"></a> [google\_groups](#output\_google\_groups) | google groups associated with the input workgroups, unqualified |
-| <a name="output_members"></a> [members](#output\_members) | authoritative, fully-qualified list of members associated with the input workgroups |
-
-<!-- BEGIN_TF_DOCS -->
-## Requirements
-
-| Name | Version |
-|------|---------|
-| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | ~> 1.0 |
-| <a name="requirement_google"></a> [google](#requirement\_google) | >= 3.0 |
-| <a name="requirement_google-beta"></a> [google-beta](#requirement\_google-beta) | >= 4.0 |
-
-## Providers
-
-No providers.
-
-## Modules
-
-| Name | Source | Version |
-|------|--------|---------|
-| <a name="module_workgroup"></a> [workgroup](#module\_workgroup) | github.com/mozilla/terraform-modules//google_workgroup | main |
-
-## Resources
-
-No resources.
+This module is cloned from https://github.com/mozilla-services/cloudops-infra-terraform-modules/tree/master/data-workgroup.
+## Example
+```hcl
+module "workgroup" {
+  source = "github.com/mozilla/terraform-modules//google_workgroup?ref=main"
```

Comment:
```suggestion
  source = "github.com/mozilla/terraform-modules//google_workgroup?ref=main"
```

This should be mozilla_workgroup reference, but this is pulled in from the example1.tf, so that needs to be updated first and then the doc regenerated.

Reply:
It looks like this was updated on the README directly, but this actually pulled in from the example via terraform-docs command using this reference https://github.com/mozilla/terraform-modules/blob/d8c958ff41991450d897ca6cd40ab25fdda87bb5/mozilla_workgroup/.terraform-docs.yml . 

We should update https://github.com/mozilla/terraform-modules/blob/d8c958ff41991450d897ca6cd40ab25fdda87bb5/mozilla_workgroup/examples/example1.tf. 

Once example1.tf is updated, run `terraform-docs` to generate the README document.

Reply:
oh I accepted the changes on the README .. I will update the example1.tf